src/main/kotlin/no/nav/familie/ef/iverksett/økonomi/simulering/SimuleringService.kt

```diff
@@ -64,4 +68,22 @@ class SimuleringService(
             oppsummering = simuleringsresultatDto
         )
     }
+
+    private fun hentSimuleringsresultatOgFiltrerPosteringer(
+        utbetalingsoppdrag: Utbetalingsoppdrag,
+        stønadType: StønadType
+    ): DetaljertSimuleringResultat {
+        val fagOmrådeKoder = fagområdeKoderForPosteringer(stønadType)
+        val simuleringsResultat = oppdragKlient.hentSimuleringsresultat(utbetalingsoppdrag)
+        return simuleringsResultat.copy(
+            simuleringsResultat.simuleringMottaker
+                .map {
+                    it.copy(
```

Comment:
Litt mange `it` her, går det å bruke et navn for noen av de, eller trekke ut til egen metode?

Reply:
Rename den indre iteratoren kanskje ? Eller trekke ut ?